pd/src/state.rs

```diff
@@ -310,6 +336,8 @@ INSERT INTO blobs (id, data) VALUES ('gc', $1)
             // `i64` in sqlx. try_into uses the `TryFrom<i64>` implementation for voting power from
             // Tendermint, so will return an error if voting power is negative (and not silently
             // overflow).
+            // TODO the voting power is actually stored on the `validator_rates` table now so
+            // we need to join against that table
```

Comment:
non-blocking: later, could this method be refactored so that it can get all the data it needs out of a single database query, rather than having to do multiple queries? (cc @aubrey-dyer)